clab/destroy.go

```diff
@@ -0,0 +1,77 @@
+package clab
```

Comment:
destroy also almost entirely unchanged (if not entirely) and just moved from clab.go